app/interactions/pulitzer/content_elements_controller/update.rb

```diff
@@ -0,0 +1,21 @@
+class Pulitzer::ContentElementsController::Update
+
+  def initialize(content_element, params)
+    @content_element, @params = content_element, params.to_h.to_hash.symbolize_keys
```

Comment:
@darkside `.to_hash` creates a new Hash object. The params are of type ActionController::Parameters, so we can't be editing that object, since it's a different class.